server/src/main/java/org/apache/uniffle/server/ShuffleServer.java

```diff
@@ -142,6 +142,7 @@ public void start() throws Exception {
     initMetricsReporter();
 
     registerHeartBeat.startHeartBeat();
+    directMemoryUsageReporter.start();
```

Comment:
Could add test unit case to test the funcation of directMemoryUsageReporter?
Or I could try later.
@rickyma